projects/plugins/jetpack/_inc/lib/core-api/class.jetpack-core-api-module-endpoints.php

```diff
@@ -1616,41 +1734,49 @@ public function get_vaultpress_data() {
 
 		$vaultpress = new VaultPress();
 		if ( ! $vaultpress->is_registered() ) {
-			return rest_ensure_response( array(
-				'code'    => 'not_registered',
-				'message' => esc_html__( 'You need to register for VaultPress.', 'jetpack' )
-			) );
-		}
-
-		$data = json_decode( base64_decode( $vaultpress->contact_service( 'plugin_data' ) ) );
-		if ( false == $data ) {
-			return rest_ensure_response( array(
-				'code'    => 'not_registered',
-				'message' => esc_html__( 'Could not connect to VaultPress.', 'jetpack' )
-			) );
-		} else if ( is_wp_error( $data ) || ! isset( $data->backups->last_backup ) ) {
+			return rest_ensure_response(
+				array(
+					'code'    => 'not_registered',
+					'message' => esc_html__( 'You need to register for VaultPress.', 'jetpack' ),
+				)
+			);
+		}
+
+		$data = json_decode( base64_decode( $vaultpress->contact_service( 'plugin_data' ) ) ); // phpcs:ignore WordPress.PHP.DiscouragedPHPFunctions.obfuscation_base64_decode
+		if ( false === $data ) {
+			return rest_ensure_response(
+				array(
+					'code'    => 'not_registered',
+					'message' => esc_html__( 'Could not connect to VaultPress.', 'jetpack' ),
+				)
+			);
+		} elseif ( is_wp_error( $data ) || ! isset( $data->backups->last_backup ) ) {
 			return $data;
-		} else if ( empty( $data->backups->last_backup ) ) {
-			return rest_ensure_response( array(
-				'code'    => 'success',
-				'message' => esc_html__( 'VaultPress is active and will back up your site soon.', 'jetpack' ),
-				'data'    => $data,
-			) );
+		} elseif ( empty( $data->backups->last_backup ) ) {
+			return rest_ensure_response(
+				array(
+					'code'    => 'success',
+					'message' => esc_html__( 'VaultPress is active and will back up your site soon.', 'jetpack' ),
+					'data'    => $data,
+				)
+			);
 		} else {
-			return rest_ensure_response( array(
-				'code'    => 'success',
-				'message' => esc_html(
-					sprintf(
-						/* translators: placeholder is a unit of time (1 hour, 5 days, ...) */
-						esc_html__( 'Your site was successfully backed up %s ago.', 'jetpack' ),
-						human_time_diff(
-							$data->backups->last_backup,
-							current_time( 'timestamp' )
+			return rest_ensure_response(
+				array(
+					'code'    => 'success',
+					'message' => esc_html(
+						sprintf(
+							/* translators: placeholder is a unit of time (1 hour, 5 days, ...) */
+							esc_html__( 'Your site was successfully backed up %s ago.', 'jetpack' ),
+							human_time_diff(
+								$data->backups->last_backup,
+								current_time( 'timestamp' ) // phpcs:ignore WordPress.DateTime.CurrentTimeTimestamp.Requested -- We have to keep companing between "WP" timestamps here in human_time_diff, we cannot switch to time() or another "unix" timestamp option.
```

Comment:
Am I right in thinking that the reason for this is because `$data->backups->last_backup` is a "WP" timestamp so we need a "WP" timestamp as the other argument to `human_time_diff()`? If so, should we say that instead? If not, what's the real reason?

As written this implies that `human_time_diff()` cares whether it gets "WP" timestamps or Unix timestamps, when in practice it doesn't care (but both have to be the same kind or it'll give wrong results).

If nothing else, "companing" seems to be a typo.

Reply:
Yeah, that's what I meant. I guess I wasn't clear enough. And this was indeed meant to be "comparing". I've tried to clarify a bit in a078aacc06e217a24c2cd502bbf31d5fda7a97bf